gitpod-io/website
# nit-fix: typo in homepage

## Description
fixes a typo on the homepage

## Related Issue(s)

Fixes #2675 

## How to test
- Open Deployed Preview URL, and scroll down on the homepage.

You will see the following: 
![image](https://user-images.githubusercontent.com/55068936/187856604-3926f347-f8bb-417c-b6a3-d8efde915e0c.png)


## Release Notes

```release-note
NONE
```

## Documentation

No


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2676"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
